rotkehlchen/exchanges/coinbase.py

```diff
@@ -660,19 +660,25 @@ def _query_single_account_transactions(
                         f'expected structure {transaction}',
                     )
             elif tx_type in ('buy', 'sell', 'advanced_trade_fill'):
-                if (trade := self._process_coinbase_trade(event=transaction)):
+                if (trade := self._process_coinbase_trade(event=transaction)) is not None:
                     trades.append(trade)
             elif (
-                    tx_type in ('interest', 'inflation_reward', 'staking_reward') or
+                    tx_type in (
+                        'interest', 'inflation_reward', 'staking_reward',
+                        'staking_transfer', 'unstaking_transfer', 'cardbuyback',
+                    ) or
                     (
-                        tx_type == 'send' and 'from' in transaction and
+                        tx_type == 'tx' and 'from' in transaction and
```

Comment:
a little context here. `tx` is the new `send`
![CleanShot 2025-01-30 at 16 16 13@2x](https://github.com/user-attachments/assets/3f7cae97-e056-4e0e-a1f3-51bc49bba089)